### PR TITLE
Harden Vaultwarden installation

### PR DESCRIPTION
**Pull request by Virviil**
## Changes
- Make Vaultwarden installation to use Postgres as storage, that allows to perform backups using standard Coolify mechanisms.
- Changing logo from licensed Bitwarden to Vaultwarden

### CODE REVIEW

**Comment by Virviil** [comment]
I don't understand how `templates/service-templates.json` file is created, but I hope that automatically.
It's probably should be changed before merging this PR.

@peaklabs-dev hope you can point me out

**Comment by peaklabs-dev** [comment]
The json will be generated by use for now. It is great that you did not commit it to the PR as that could cause problems. You van test your changes using the docker compose empty deploy type.

**Comment by Virviil** [comment]
@peaklabs-dev I've checked it as docker-compose deployment - all is working (as expected for this setup)

File: templates/compose/vaultwarden.yaml [hunk 1]
```diff
@@ -1,26 +1,62 @@
...
-# logo: svgs/bitwarden.svg
+# logo: svgs/vaultwarden.svg
 # port: 80
 
 services:
+  # Using PG database to allow backups.
+  # Everything EXCEPT *sends* will be restored from backup even on Coolify recreation.
+  postgres:
+    # Using alpine image to reduce size
+    image: postgres:alpine
```
Comment (peaklabs-dev) [inline templates/compose/vaultwarden.yaml:12]: Please use a version number such as 16 or so.

File: templates/compose/vaultwarden.yaml [hunk 2]
```diff
@@ -1,26 +1,62 @@
...
       - DOMAIN=${SERVICE_FQDN_VAULTWARDEN}
-      - DATABASE_URL=${VAULTWARDEN_DB_URL:-data/db.sqlite3}
-      - SIGNUPS_ALLOWED=${SIGNUP_ALLOWED:-true}
+      # DB Config
+      - DATABASE_URL=postgresql://${SERVICE_USER_POSTGRES}:${SERVICE_PASSWORD_POSTGRES}@postgres:5432/${SERVICE_DB_POSTGRES:-vaultwarden}
+      # Admin token is optional - if not set, the admin panel will be disabled.
+      # To disable the admin panel, unset this.
+      # You probably should do this for security reasons.
       - ADMIN_TOKEN=${SERVICE_PASSWORD_64_ADMIN}
-      - IP_HEADER=X-Forwarded-For
```
Comment (peaklabs-dev) [inline templates/compose/vaultwarden.yaml:16]: Keep this variable as it is needed to get the right IPs

**Review by peaklabs-dev — CHANGES_REQUESTED** [review]
Since the postgres version is like a new version of vaultwarden, add a `vaultwarden-postgres.yaml` file and leave the old one as it is. 

Also, I have tested it and it does not work for me, so please go through the new template and test it thoroughly.

**Comment by peaklabs-dev** [comment]
Thank you for your contribution. Due to inactivity, we're closing this PR. If you'd like to revisit this change, feel free to reopen this PR or open a new pull request.

**Pull request closed without merging**